src/common/_buttons.scss

```diff
@@ -173,6 +173,11 @@
       background-color: $close-button-active-background-color;
     }
   }
+
+  &[disabled] {
+    filter: grayscale(1.0) opacity(0.7);
+    cursor: not-allowed;
```

Comment:
Not sure, whether cursor: not-allowed (show forbidden cursor) or pointer-events: none is better. I prefer the latter, but are there any accessibility considerations?

Reply:
`pointer-events: none` is already done for you on disabled buttons by the browser. `cursor: not-allowed` is the visual indication that clicking on it will do nothing.